### PR TITLE
DocBlock rework

### PR DESCRIPTION
**Pull request by florianbrinkmann**
Adds the `@change` tags back that were replaced by `2.10.0` and replaces `@change` with `@since`.

### CODE REVIEW

File: antispam_bee.php [hunk 1]
```diff
@@ -407,8 +409,8 @@ public static function deactivate() {
 	/**
 	 * Action deleting the plugin
 	 *
-	 * @since   2.4
-	 * @change  2.4
+	 * @since  2.4
+	 * @since  2.4
```
Comment (pfefferle) [inline antispam_bee.php:413]: I think one `@since 2.4` is enough ;)

File: antispam_bee.php [hunk 2]
```diff
@@ -583,8 +586,8 @@ public static function load_plugin_lang() {
 	/**
 	 * Add the link to the settings
 	 *
-	 * @since   1.1
-	 * @change  1.1
+	 * @since  1.1
+	 * @since  1.1
```
Comment (pfefferle) [inline antispam_bee.php:590]: duplicate `@since`

File: antispam_bee.php (hunk 1 above)
Comment (florianbrinkmann) [reply antispam_bee.php]: thanks! :)

**Review by pfefferle — APPROVED** [review]
LGTM

**Pull request merged**